packages/jupytercad-extension/src/mainview.tsx

```diff
@@ -1065,11 +1064,8 @@ export class MainView extends React.Component<IProps, IStates> {
   };
 
   private _handleWindowResize = (): void => {
-    clearTimeout(this._resizeTimeout);
-    this._resizeTimeout = setTimeout(() => {
-      this.forceUpdate();
-      this._updateAnnotation({ updatePosition: true });
-    }, 500);
```

Comment:
We don't need debouncing anymore?

Reply:
I suppose the debouncing was because `forceUpdate` was CPU/GPU intensive, but we don't need to call forceUpdate IMO.

The window resize is way smoother with these changes, it's immediate.